style.css

```diff
@@ -76,14 +69,239 @@ h2 {
   color: #7f8cff;
 }
 
+/* Works section */
+.works-container,
+.multi-post-container {
+  display: grid;
+  grid-template-rows: auto;
+  justify-content: center;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #dfe1e6;
+}
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the Correct `font-family`, `font-color` and etc. which makes your design `inconsistent` and looks unprofessional. Please use equal padding and have a look at the below-attached image and table. 

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-06-10 17-48-23](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/362f3cd9-d109-40ce-8c88-877085589882)

</td>
<td>

![Screenshot from 2023-06-10 17-48-56](https://github.com/Otromascomotu/My-Portfolio-setup-and-mobile-first/assets/66476329/5ba1dd4b-fdb8-499d-bbab-6d2803664042)

</td>
</tr>
</table>

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)